### ISSUE TITLE
dhcp client

### ISSUE THREAD

**Issue by ipartner** [krolaw/dhcp4#20]
Hi.
There is a way to create a dhcp client with this library?

**Comment by michaelharo** [comment]
yes - Line 151 of packet.go says "// Creates a request packet that a Client would send to a server."

**Comment by d2g** [comment]
I've been working on a client package (https://github.com/d2g/dhcp4client) which adds some wrappers to the functionality provided by this package.

**Comment by ipartner** [comment]
Nice, I will look at it.
